WalletWasabi.Fluent/Views/HomePageView.axaml

```diff
@@ -2,14 +2,46 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:vm="clr-namespace:WalletWasabi.Fluent.ViewModels"
              xmlns:c="clr-namespace:WalletWasabi.Fluent.Controls"
-             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"             
+             xmlns:conv="clr-namespace:WalletWasabi.Fluent.Converters"
+             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+             x:DataType="vm:HomePageViewModel"
              x:Class="WalletWasabi.Fluent.Views.HomePageView">
-  <c:ContentArea Title="Home" Caption="Select a Wallet below to continue or create a new one:">
-    <StackPanel HorizontalAlignment="Center"
-              VerticalAlignment="Center">
-      <TextBlock Text="Hi, I'm the first view!" />
-      <TextBlock Text="{Binding UrlPathSegment}" />
-    </StackPanel>
+  <c:ContentArea Title="Home">
+    <c:ContentArea.Caption>
+      <DockPanel>
+        <Button Padding="0" Classes="invisible" DockPanel.Dock="Right" Command="{Binding OpenWalletsFolderCommand}">
```

Comment:
The Open Wallets folder could have more padding so when pointer is over highlight is bigger than text

Reply:
removed pointer over styling here...

hoping to remove this button all together in future.